arcaflow_plugin_sysbench/sysbench_plugin.py

```diff
@@ -120,9 +117,11 @@ def RunSysbenchCpu(
 
     print("==>> Running sysbench CPU workload ...")
 
-    cpu_flags = [
-        "--cpu-max-prime=" + str(params.cpu_max_prime),
-    ]
+    serialized_params = sysbench_cpu_input_schema.serialize(params)
```

Comment:
I question whether there are any gotchas with this method. It also lowers flexibility to handle special cases.

Reply:
The idea here is to limit as much as possible being opinionated, and let the schema itself speak for the command inputs. Without this approach, anything added to the schema input also needs to have a separate handler written into the plugin.

Let me know if you have any specific thoughts about the flexibility limitations we might need to worry about.